src/components/shared/card.js

```diff
@@ -36,7 +40,34 @@ import Spike from '../../icons/weapons/talents/Spike.png'
 import SteadyHanded from '../../icons/weapons/talents/Steady Handed.png'
 import Strained from '../../icons/weapons/talents/Strained.png'
 import Vindictive from '../../icons/weapons/talents/Vindictive.png'
+// Brand Sets
+import FiveOneOne from '../../icons/gear_sets/511.png';
```

Comment:
Is there a way to majorly condense the imports or naw?

Reply:
Not that I know of. I did some research and found people using `require.context` but I don't know if that will work the same.